x-pack/plugins/security_solution_serverless/public/get_started/card_step/index.tsx

```diff
@@ -12,78 +12,133 @@ import {
   EuiIcon,
   EuiBadge,
   useEuiTheme,
-  EuiButtonEmpty,
-  useEuiBackgroundColorCSS,
+  EuiButtonIcon,
+  useEuiBackgroundColor,
 } from '@elastic/eui';
 import { css } from '@emotion/react';
-import React, { useCallback, useMemo } from 'react';
+import React, { useCallback, useEffect, useMemo } from 'react';
 
 import classnames from 'classnames';
-import type { CardId, OnStepButtonClicked, OnStepClicked, SectionId, StepId } from '../types';
-import icon_step from '../images/icon_step.svg';
-import icon_cross from '../images/icon_cross.svg';
-import { UNDO_MARK_AS_DONE_TITLE, MARK_AS_DONE_TITLE } from '../translations';
+import { APP_UI_ID, SecurityPageName } from '@kbn/security-solution-plugin/common';
```

Comment:
It's better to import the `SecurityPageName` from the package directly:
```suggestion
import { useNavigateTo, SecurityPageName } from '@kbn/security-solution-navigation';
```
Also, the `APP_UI_ID` is not needed using `useNavigateTo`, example:
```
  const navigateTo = useNavigateTo();
  [...]
  navigateTo({
    deepLinkId: SecurityPageName.landing,
    path: `#${stepId}`,
  });
```